operator/webhooks/pod_mutating_webhook.go

```diff
@@ -297,12 +297,12 @@ func (a *PodMutatingWebhook) generateWorkload(ctx context.Context, pod *corev1.P
 		postDeploymentTasks = strings.Split(pod.Annotations[common.PostDeploymentTaskAnnotation], ",")
 	}
 
-	if pod.Annotations[common.PreDeploymentAnalysisAnnotation] != "" {
-		preDeploymentAnalysis = strings.Split(pod.Annotations[common.PreDeploymentAnalysisAnnotation], ",")
+	if pod.Annotations[common.PreDeploymentEvaluationAnnotation] != "" {
```

Comment:
here we can align the retrieval of the annotations to the changes made in https://github.com/keptn-sandbox/lifecycle-controller/pull/181